update-DOM.js

```diff
@@ -0,0 +1,55 @@
+const totalSpan = document.getElementById('total');
+const percentSpan = document.getElementById('percent');
+const lossesSpan = document.getElementById("losses");
+const winsSpan = document.getElementById('wins');
+const tied = document.getElementById('tied');
+const results = document.getElementById('results');
+const resetSpan = document.getElementById('reset-counter');
+
+// initialize state
+let totalWins = 0;
+let gamesPlayed = 0;
+let gamesTied = 0;
+
+// 
+let reset = 0;
+
+const losses = () => (gamesPlayed - totalWins);
```

Comment:
Your losses when you play the game, is incrementing so that your draws are also incrementing.  I think instead of using this function and calling it on line 22, you should treat it like your totalWins and and gamesTied instead.